debian/changelog

```diff
@@ -0,0 +1,5 @@
+python-ovh (0.0.1-1) trusty; urgency=low
+
+  * Initial release
+
+ -- Arnaud Morin <arnaud.morin@corp.ovh.com>  Fri, 30 Sep 2016 11:56:51 +0200
```

Comment:
I'm afraid that's ``jean-tiare.le-bigot@corp.ovh.com`` here unless you plan to manage it in the long run ^^ (oetherwise, you'll end up with tons of "NON MAINTAINER UPLOAD" warnings everywhere)